README.md

```diff
@@ -12,36 +12,25 @@ This repository contains the Sample service. This microservice is a RESTful web
 * Receiving confirmation from the Party service once it has consumed/stored the sample units sent above, and marking the sample summary as ready for collection
 * Providing on request the sample summaries matching a given criteria to the Collection Exercise service
 
-## Prerequisites
+## Running
 
-* To run this project you must have the following running:
+There are two ways of running this service
 
-    - rabbitMQ
-    - redis
-    - postgres
-    - an sftp server
-
-  You can get all of this using the following Docker project:
-
-    https://github.com/ONSdigital/rm-docker-dev
-
-* You must first clone and build the following projects:
-
-    - https://github.com/ONSdigital/rm-common-service
-    - https://github.com/ONSdigital/rm-party-service-api
-
-  and build them using
-
-```bash
-mvn clean install
-```
+* The easiest way is via docker (https://github.com/ONSdigital/ras-rm-docker-dev)
+* Alternatively running the service up in isolation
+    ```bash
+    cp .maven.settings.xml ~/.m2/settings.xml  # This only needs to be done once to set up mavens settings file
```

Comment:
I can't find a `.maven.settings.xml` file in this repo?

Reply:
Hi @benjefferies where is the .maven.settings.xml file?

Reply:
Good spot guys. I messed something up when committing. I've resolved it though